webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria.js

```diff
@@ -419,6 +419,13 @@ export class SakaiRubricCriteria extends RubricsElement {
     .done(() => this.letShareKnow())
     .fail((jqXHR, error, message) => {console.log(error); console.log(message);});
   }
+
+ openEditWithKeyboard(e){
+    if(e.keyCode == 32 || e.keyCode == 32 ){
```

Comment:
Did you mean to check two codes here? And, could you leave a space after the if, and one before the opening brace?